config/routes.rb

```diff
@@ -14,18 +14,8 @@
     get 'timeout' => 'users/sessions#timeout'
   end
 
-  get '/users/service_providers' => 'users/service_providers#index', as: 'users_service_providers'
-  post '/users/service_providers' => 'users/service_providers#create'
-  get '/users/service_providers/new' => 'users/service_providers#new',
-      as: 'new_users_service_provider'
-  get '/users/service_providers/:id/edit' => 'users/service_providers#edit',
-      as: 'edit_users_service_provider'
-  get '/users/service_providers/:id' => 'users/service_providers#show', as: 'users_service_provider'
-  patch '/users/service_providers/:id' => 'users/service_providers#update'
-  put '/users/service_providers/:id' => 'users/service_providers#update'
-  delete '/users/service_providers/:id' => 'users/service_providers#destroy'
-
-  resources :organizations, only: [:new, :create, :show, :edit, :update, :destroy, :index]
```

Comment:
I really like the `only: []` syntax for highlighting which routes actually get used. WDYT of keeping it? It also makes `rake routes` cleaner because it doesn't show the ones we don't use